objectrocket/instances/mongodb.py

```diff
@@ -155,3 +170,8 @@ def _get_connection(self, ssl):
             connect_string = self.ssl_connect_string
 
         return pymongo.MongoClient(connect_string)
+
+    @property
+    def _stats_url(self):
+        """The URL for stats of this instance."""
+        return self._instances._url + '{}/{}/'.format(self.name, 'new-relic-stats')
```

Comment:
Take a gander at this guy: https://github.com/objectrocket/python-client/blob/master/objectrocket/bases.py#L277-L280.
You can use that inherited property